lib/robots/viomi/capabilities/ViomiZoneCleaningCapability.js

```diff
@@ -0,0 +1,57 @@
+const ZoneCleaningCapability = require("../../../core/capabilities/ZoneCleaningCapability");
+const ViomiMapParser = require("../../../ViomiMapParser");
+const Logger = require("../../../Logger");
+
+class ViomiZoneCleaningCapability extends ZoneCleaningCapability {
+    /**
+     * @param {Array<import("../../../entities/core/ValetudoZone")>} valetudoZones
+     * @returns {Promise<void>}
+     */
+    async start(valetudoZones) {
+        let areas = [];
+
+        valetudoZones.forEach( zone => {
+            const pA = ViomiMapParser.positionToViomi(zone.points.pA.x, zone.points.pA.y);
+            const pB = ViomiMapParser.positionToViomi(zone.points.pB.x, zone.points.pB.y);
+            const pC = ViomiMapParser.positionToViomi(zone.points.pC.x, zone.points.pC.y);
+            const pD = ViomiMapParser.positionToViomi(zone.points.pD.x, zone.points.pD.y);
+
+            for (let j = 0; j < zone.iterations; j++) {
+                areas.push([areas.length,
+                    0,
```

Comment:
use named constants. or at least document the meaning: 0: regular zone, 2: restricted area

Reply:
done